src/components/views/PreLoader/PreLoader.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
```

Comment:
I think it is better to use external packages for loader (ex https://github.com/CognizantStudio/react-loader)  instead of implemented by yourself